test/csuite/wt2999_join_extractor/main.c

```diff
@@ -86,8 +86,9 @@ main(int argc, char *argv[])
     WT_CURSOR *cursor1, *cursor2, *jcursor;
     WT_ITEM k, v;
     WT_SESSION *session;
-    int32_t key, val[2];
-    int i, ret;
+    /* To avoid integer overflow we have to use uint32_t instead of int32_t for key and value. */
+    uint32_t i, key, val[2];
```

Comment:
You're still going past the max value for an `uint32_t` in the key and value. They are just wrapping around due to the unsigned.

Reply:
You're right thanks!

Reply:
Nope, but ideally we'd not do this intentionally :)